### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by jessica440** [fac20/Week-2-GMNO#41]
- [x] for better accessibility the form should have associated labels.
- [ ] it would be great if the tab would select the different cards (at the moment it only selects the search bar and button)
